rio_cogeo/cogeo.py

```diff
@@ -354,27 +351,6 @@ def cog_translate(  # noqa: C901
                         )
 
                     dst_kwargs["driver"] = "COG"
-                    if web_optimized:
-                        dst_kwargs["TILING_SCHEME"] = (
-                            "GoogleMapsCompatible"
-                            if tms.identifier == "WebMercatorQuad"
-                            else tms.identifier
```

Comment:
there was no guarantee that `tms.identifier` would be compatible with GDAL TMS, and in `TMS 2.0` the `id` is now optional so we can't use it